src/wp-admin/includes/class-wp-upgrader.php

```diff
@@ -798,6 +798,15 @@ public function run( $options ) {
 			)
 		);
 
+		/**
+		 * Filter result of install_package().
```

Comment:
```suggestion
		 * Filters the result of install_package().
```

https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#language